carma_types_mod.F90

```diff
@@ -400,9 +400,9 @@ module carma_types_mod
     logical                                       :: f_do_cnst_rlh
     logical, allocatable, dimension(:,:)          :: f_if_nuc       !(NELEM,NELEM)
     real(kind=f)                                  :: f_conmax
-    integer                                       :: f_igash2o
-    integer                                       :: f_igash2so4
-    integer                                       :: f_igasso2
+    integer                                       :: f_igash2o = 0
```

Comment:
Why are these being initialized here? This doesn't happen for any other field. This is just meant to by the type definition. Initialization occurs within the object. In this case it is the carma object, so they should be initialized in carma_mod.F90. It current happens in CARMA_InitializeGrowth()

    ! Initialize indexes for the gases and check to make sure if H2SO4 is used
    ! that it occurs after H2O. This is necessary for supersaturation calculations.
    carma%f_igash2o   = 0
    carma%f_igash2so4 = 0
    carma%f_igasso2   = 0

I don't think these fields should be accessed unless do_growth is true, so if they are then we should double check that. Otherwise if they need to be initialized all the time, they could be set to 0 in CARMA_Initialize().